.github/workflows/update-flatpak-docs.yml

```diff
@@ -0,0 +1,58 @@
+name: Update Flatpak and Flatpak Builder docs
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
+  workflow_dispatch: {}
+
+jobs:
+  archive:
+    name: Update Flatpak and Flatpak Builder docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: |
+          sudo add-apt-repository ppa:flatpak/development
+          sudo apt update
+          sudo apt install -y flatpak flatpak-builder git libxml2-utils
+
+      - name: Get current docs version
+        run: |
+          old_flatpak_version="$(xmllint --html --xpath "//p[@class='releaseinfo']/text()" docs/flatpak-docs.html|sed "s/Version//g"|awk '{$1=$1};1')"
+          old_flatpak_builder_version="$(xmllint --html --xpath "//p[@class='releaseinfo']/text()" docs/flatpak-builder-docs.html|sed "s/Version//g"|awk '{$1=$1};1')"
+          echo "OLD_FLATPAK_VERSION=${old_flatpak_version}" >> $GITHUB_ENV
+          echo "OLD_FLATPAK_BUILDER_VERSION=${old_flatpak_builder_version}" >> $GITHUB_ENV
+
+      - name: Move docs
+        run: |
+          cp -vf /usr/share/doc/flatpak/flatpak-docs.html docs/
+          cp -vf /usr/share/doc/flatpak-builder/flatpak-builder-docs.html docs/
+
+      - name: Get new docs version
+        run: |
+          new_flatpak_version="$(xmllint --html --xpath "//p[@class='releaseinfo']/text()" docs/flatpak-docs.html|sed "s/Version//g"|awk '{$1=$1};1')"
+          new_flatpak_builder_version="$(xmllint --html --xpath "//p[@class='releaseinfo']/text()" docs/flatpak-builder-docs.html|sed "s/Version//g"|awk '{$1=$1};1')"
+          echo "NEW_FLATPAK_VERSION=${new_flatpak_version}" >> $GITHUB_ENV
+          echo "NEW_FLATPAK_BUILDER_VERSION=${new_flatpak_builder_version}" >> $GITHUB_ENV
+
+      - name: Compare versions
+        run: |
+          appstreamcli vercmp $NEW_FLATPAK_VERSION gt $OLD_FLATPAK_VERSION
+          appstreamcli vercmp $NEW_FLATPAK_BUILDER_VERSION gt $OLD_FLATPAK_BUILDER_VERSION
```

Comment:
So this will only succeed if both get updates, they don't have matched release schedules.

Reply:
Can separate it out to two schedules.